.spelling

```diff
@@ -135,6 +135,10 @@ CNs
 color
 colored
 colors
+config
```

Comment:
I don't think we would want config (and its variants) in the spelling list. It seems like any time it is being used it should either be expanded to configuration or, if it's talking about a specific command/term on a switch or system, then we'd want it in backticks.